.github/workflows/codeql-analysis.yml

```
@@ -13,24 +13,28 @@ name: 'CodeQL'
 
 on:
   push:
-    branches: [main, 0.x]
+    branches: [main, 0.x.1]
   pull_request:
     # The branches below must be a subset of the branches above
     branches: [main]
   schedule:
-    - cron: '27 0 * * 0'
+    - cron: '24 0 * * 1 8'
```

actionlint (1.7.12)

[error] invalid CRON format "24 0 * * 1 8" in schedule event: expected exactly 5 fields, found 6: [24 0 * * 1 8]

(events)